main/wget/APKBUILD

```diff
@@ -1,15 +1,16 @@
+# Contributor: Sandro Jäckel <sandro.jaeckel@posteo.de>
 # Contributor: Sergei Lukin <sergej.lukin@gmail.com>
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
 # Maintainer: Carlo Landmeter <clandmeter@gmail.com>
 pkgname=wget
 pkgver=1.20.3
-pkgrel=0
+pkgrel=1
 pkgdesc="A network utility to retrieve files from the Web"
 url="https://www.gnu.org/software/wget/wget.html"
 arch="all"
 license="GPL-3.0-or-later"
-depends=""
-makedepends="openssl-dev perl"
+depends="libc6-compat"
```

Comment:
I build it with `libintl` dependency instead of that and it works for me

Reply:
Building it with ``libintl`` does not work.
````
configure: WARNING: *** libpsl was not found. Fallback to builtin cookie checking.
configure: WARNING: *** LIBIDN2 was not found. You will not be able to use IDN2008 support
````
At runtime it does work but libintl is way bigger than libc6-compat